### ISSUE TITLE
Turbolink clicks not announced

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#302]
2 observed issues:

1. clicking a subject link from an individual record
2. selecting an option in a facet

_from accessibility testing with Michelle on 5/14/19_

**Comment by ajkiessl** [comment]
I'm not too sure what the expected behavior is here.  When I use Mac VoiceOver the subject and facet links behave like any other link.

**Comment by ajkiessl** [comment]
Confirmed with Michelle that this is no longer an issue, so closing.  We did, however, find another issue: #1190 .
